controllers/app.controller.js

```diff
@@ -0,0 +1,10 @@
+const { readTopics } = require("../models/app.model");
+
+function getTopics(req, res, next) {
+  readTopics()
+    .then((topics) => {
+      res.status(200).send(topics);
```

Comment:
We should send back data to the client on a meaningful key e.g, they should be able to access it on body.endpoints :)